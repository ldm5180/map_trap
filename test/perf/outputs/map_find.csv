Run on (4 X 2594.09 MHz CPU s)
2017-10-28 16:39:19
name,iterations,real_time,cpu_time,time_unit,bytes_per_second,items_per_second,label,error_occurred,error_message
"BM_map_find<std::map<int, std::string>>/8",10067759,63.0742,63.0744,ns,,,,,
"BM_map_find<std::map<int, std::string>>/16",9291247,74.2499,74.2501,ns,,,,,
"BM_map_find<std::map<int, std::string>>/32",7639044,90.4937,90.4936,ns,,,,,
"BM_map_find<std::map<int, std::string>>/64",5602796,122.607,122.606,ns,,,,,
"BM_map_find<std::map<int, std::string>>/128",4164010,187.861,187.695,ns,,,,,
