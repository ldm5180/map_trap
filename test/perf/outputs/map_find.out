Run on (4 X 2594.09 MHz CPU s)
2017-10-28 16:27:43
name,iterations,real_time,cpu_time,time_unit,bytes_per_second,items_per_second,label,error_occurred,error_message
"BM_map_find<std::map<int, std::string>>/8",11055601,61.6531,61.6805,ns,,,,,
"BM_map_find<std::map<int, std::string>>/16",10181525,71.3213,71.3548,ns,,,,,
"BM_map_find<std::map<int, std::string>>/32",7214938,89.7759,89.8174,ns,,,,,
"BM_map_find<std::map<int, std::string>>/64",5777515,119.475,119.523,ns,,,,,
"BM_map_find<std::map<int, std::string>>/128",4286771,167.79,167.821,ns,,,,,
